docs/auth/federated-auth.md

```diff
@@ -195,7 +195,7 @@ with the Facebook App ID and Secret set.
   }
   ```
 
-Note: Firebase will not set the `User.emailVerified` property
+Note: Firebase will not set the `User.isEmailVerified` property
```

Comment:
.emailVerified is deprecated but still available. It's changed to match Java and the isAnonymous